docs/Simorgh-Release-Info.mdx

```diff
@@ -19,7 +19,7 @@ Remember to commit and push the changes to Simorgh-Release-Info.md -->
-| marathi | [local: Article](http://localhost:7080/marathi/articles/cp47g4myxz7o)<br/>[local: FIX](http://localhost:7080/marathi/india-53901688)<br/>[local: Home](http://localhost:7080/marathi)<br/>[local: MAP](http://localhost:7080/marathi/media-23127353)<br/>[local: Most Read](http://localhost:7080/marathi/popular/read)<br/>[local: Most Watched](http://localhost:7080/marathi/media/video)<br/>[local: Podcast (Brand)](http://localhost:7080/marathi/podcasts/p09431p4)<br/>[local: Podcast (Episode)](http://localhost:7080/marathi/podcasts/p09431p4/p09bplch)<br/>[local: PGL](http://localhost:7080/marathi/india-42894522)<br/>[local: Topic](http://localhost:7080/marathi/topics/c5qvpxvv7y3t) | [test: Article](https://www.test.bbc.com/marathi/articles/cp47g4myxz7o)<br/>[test: FIX](https://www.test.bbc.com/marathi/23195696)<br/>[test: Home](https://www.test.bbc.com/marathi)<br/>[test: MAP](https://www.test.bbc.com/marathi/media-23127353)<br/>[test: Most Read](https://www.test.bbc.com/marathi/popular/read)<br/>[test: Most Watched](https://www.test.bbc.com/marathi/media/video)<br/>[test: Podcast (Brand)](https://www.test.bbc.com/marathi/podcasts/p09431p4)<br/>[test: Podcast (Episode)](https://www.test.bbc.com/marathi/podcasts/p09431p4/p09kx7q9)<br/>[test: On Demand TV (Brand)](https://www.test.bbc.com/marathi/bbc_marathi_tv/tv_programmes/w13xttr2)<br/>[test: PGL](https://www.test.bbc.com/marathi/23247226)<br/>[test: STY](https://www.test.bbc.com/marathi/india-23126095)<br/>[test: Topic](https://www.test.bbc.com/marathi/topics/c5qvpxvv7y3t) | [live: Article](https://www.bbc.com/marathi/articles/cvjxwvn04yjo)<br/>[live: FIX](https://www.bbc.com/marathi/india-53901688)<br/>[live: Home](https://www.bbc.com/marathi)<br/>[live: MAP](https://www.bbc.com/marathi/media-51314817)<br/>[live: Most Read](https://www.bbc.com/marathi/popular/read)<br/>[live: Most Watched](https://www.bbc.com/marathi/media/video)<br/>[live: Podcast (Brand)](https://www.bbc.com/marathi/podcasts/p09431p4)<br/>[live: Podcast (Episode)](https://www.bbc.com/marathi/podcasts/p09431p4/p09kx7q9)<br/>[live: On Demand TV (Brand)](https://www.bbc.com/marathi/bbc_marathi_tv/tv_programmes/w13xttr2)<br/>[live: PGL](https://www.bbc.com/marathi/india-42894522)<br/>[live: STY](https://www.bbc.com/marathi/india-53414454)<br/>[live: Topic](https://www.bbc.com/marathi/topics/c5qvpxvv7y3t) |
```

Comment:
Updated sections: 
`[test: Podcast (Episode)](https://www.test.bbc.com/marathi/podcasts/p09431p4/p09kx7q9)`
and 
` live: Podcast (Episode)](https://www.bbc.com/marathi/podcasts/p09431p4/p09kx7q9)`